MAINTAINERS.md

```diff
@@ -0,0 +1,165 @@
+requirements listed below.
+
+### 1. Enable 2-Factor Account Authentication for GitHub
+
+See GitHub's documentation about [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) for details.
+
+### 2. Be an active contributor
+
+Active contributors are considered to be anyone who meets any of the following criteria:
+
+- Sent more than two pull requests (PRs) in the previous one month, or more than
+  20 PRs in the previous year.
+- Filed more than three issues in the previous month, or more than 30 issues in
+  the previous 12 months.
+- Commented on more than pull requests in the previous month, or more than 50
+  pull requests in the previous 12 months.
+- Marked any PR as LGTM in the previous month.
+- Have collaborator permissions in the Kubernetes GitHub organization.
+
+See [Kubernetes Community Expectations](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md) for more details.
+
+**TODO:** Move the definition of an active contributor to here or remove this
+section and link to expectations.
+
+#### Contribution types
+
+Contributions are not limited to code. Other contributions include PR reviews,
+discussion participations, test ownership, SIG leadership, documentation, etc.
+As we improve our project tooling, we'll be able to measure and analyze other
+types of contributions and take those into account when evaluating a maintainer
+application, however maintainers still need to be familiar with the design and
+implementation of Kubernetes.
+
+### 3. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+**Note:** A year time requirement will be replaced once the contributor ladders
+are introduced.
+
+### 4. Find 3 senior maintainer sponsors
+
+As you contribute to Kubernetes, you'll be collaborating with different
+community members. Having the sponsorship of a senior project maintainer means
+that they have seen your work and they believe that you'll be a good addition as
+a project maintainer.
+
+**Motivation:** We want the sponsorships to be meaningful, so we require the sponsors to be
+maintainers that collaborated with you and we also don't want to burden the
+senior maintainers otherwise they'll be reluctant to sponsor anyone. Therefore,
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer. Also
+related to contributor ladder.
+
+## Maintainership Application
+
+Once you have satisfied the listed maintainership requirements, you can submit
+your application to kubernetes-wg-contribex@googlegroups.com wherein the
```

Comment:
We should probably create a kubernetes-maintainers mailing list to use for communication among maintainers, including such applications.